src/main/kotlin/racingcar/Cars.kt

```diff
@@ -0,0 +1,26 @@
+package racingcar
+
+class Cars(var cars: List<Car>) {
+    fun move(): String {
+        var result = ""
+        cars.forEach {
+            it.move((RANDOM_NUMBER_RANGE).random())
+            result += it.makeResult()
+        }
+        return result
+    }
+
+    fun findMaxStatusValue(): Int {
+        var max = Int.MIN_VALUE
+        for (car in cars) {
+            if (car.status > max) {
+                max = car.status
+            }
+        }
+        return max
+    }
```

Comment:
```suggestion
    fun findMaxStatusValue(): Int = cars.maxOf { it.status }
```

kotlin의 `maxOf`를 사용하면 깔끔하게 리팩토링이 가능해보여요!

`maxOf`에 대해서 알아볼까요~? 

링크
- [maxOf](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.comparisons/max-of.html)